packages/playground/cli/src/mount.ts

```diff
@@ -0,0 +1,75 @@
+import { existsSync } from 'fs';
+import path from 'path';
+import { createNodeFsMountHandler } from '@php-wasm/node';
+import type { PHP } from '@php-wasm/universal';
+
+export interface Mount {
+	hostPath: string;
+	vfsPath: string;
+}
+
+/**
+ * Parse an array of mount argument strings where the host path and VFS path
+ * are separated by a colon.
+ * e.g. [ '/host/path:/vfs/path', '/host/path:/vfs/path' ]
+ *
+ * @param mounts - An array of mount argument strings separated by a colon.
+ * @returns An array of Mount objects.
+ */
+export function parseMountWithDelimiterArguments(mounts: string[]): Mount[] {
```

Comment:
@adamziel asked

Let's add at least minimal docs to explain what types of data each of this function is parsing, e.g. "Splits a colon-separated mount definition in two paths, e.g. parseMountWithDelimiterArguments(["pathA:pathB", "pathC:pathD"]) -> [{hostPath: etc....

Reply:
Done, I updated both functions with doc comments to explain what are the expected inputs are how they are parsed.